babel.config.js

```diff
@@ -2,12 +2,16 @@ module.exports = function(api) {
   api.cache(true);
   return {
     presets: [
-      ['@babel/preset-env', {
-        useBuiltIns: 'usage',
-        corejs: 3,
-        exclude: ['es.promise'],
-      }],
+      [
+        '@babel/preset-env',
+        {
+          useBuiltIns: 'usage',
+          corejs: 3,
+          exclude: ['es.promise'],
+        },
+      ],
       '@babel/preset-react',
     ],
+    plugins: ['@babel/plugin-proposal-optional-chaining', '@babel/plugin-proposal-nullish-coalescing-operator'],
```

Comment:
you did not state that this PR was dependent on #1168, please explicitly state if you have PR dependencies so we can address them one by one

Reply:
It makes the same changes, but was made prior to #1168. I made that PR since I noticed I had made the same changes on a few branches and it seemed best to pull it out.